hugo/content/posts/about.md

```diff
@@ -7,16 +7,36 @@ menu:
     other: {}
 ---
 
-`yt` is a [YAML](http://www.yaml.org/) processor in the spirit of [`jq`](https://stedolan.github.io/jq/) and [sed](https://en.wikipedia.org/wiki/Sed).
+`yt` is a [YAML](http://www.yaml.org/) processor in the spirit of [`jq`](https://stedolan.github.io/jq/) and [sed](https://en.wikipedia.org/wiki/Sed).  It can take in many yml files, process them using gomplate language, and then create a new .yml (or .json) file.
+
+```
+    +------+                                      
+    |      |                                      
+    | .yml |\                                     
+    |      | -\                                   
+    +------+   -\                                 
+                 -\                               
+    +------+       -\+-----------+        +------+
+    |      |         -           |        |      |
+    | .yml | --------|    yt     |--------| .yml |
+    |      |        -|           |        |      |
+    +------+      -/ +-----------+        +------+
+                 /                                
+    +------+   -/                                 
+    |      | -/                                   
+    | .yml |/                                     
+    |      |                                      
+    +------+                                      
+```
 
  * `yt` can work with multi-document YAML files.
- * `yt` is VERY experimental - please use with care.
+ * `yt` is experimental - please use with care.
 
 ## Goals
 
- * Query yaml documents
+  * Query yaml documents
   * Support multi-document files (with `---` separator lines)
- * Manipulate yaml documents
+  * Manipulate yaml documents
```

Comment:
This indentation was intentional. Maybe subheadings would have been better

Reply:
Whoops sorry!